src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLambdaExpressions.java

```diff
@@ -11,19 +11,21 @@
 public class InputWhitespaceAfterLambdaExpressions {
 
     static Runnable r1 = () -> { // ok
-        String.valueOf("Hello world one!");
+        // String.valueOf("Hello world one!");
     };
 
-    static Runnable r2 = () ->String.valueOf("Hello world two!"); // violation
+    static Runnable r2 = () ->String.valueOf("Hello world two!"); // violation ''->' .* whitespace'
```

Comment:
Please use a clearer violation message, see https://github.com/checkstyle/checkstyle/pull/11342#discussion_r815335679

If line length doesn't permit, shorten variable names, remove non-test-related modifiers, change indentation (You may reduce it but keep it uniform), wrap it onto the next line and try to squeeze it in somehow. If nothing is working, you can replace it with a similar case that fits well.

The last resort will be to use `// violation above/below` pattern.

Reply:
@Vyom-Yadav can you please help me more to solve this I have tried to change some indentation but still line is too long and I have try to wrap the line onto the next line but I cant figure it out

Reply:
> shorten variable names, remove non-test-related modifiers

Did you try it?

What difficulties are you facing? There is no need to modify indentation when it can simply be modified as-
`Runnable r = () ->String.valueOf(""); // violation ''->' is not followed by whitespace'` which should be under the limit of 100.

Reply:
Actually I have tried this same but in line 19 it would still getting error so i think to remove some indentation

Reply:
Resolved.